core/src/main/java/org/infinispan/reactive/publisher/impl/commands/batch/NextPublisherCommand.java

```diff
@@ -14,8 +14,6 @@
 public class  NextPublisherCommand extends BaseRpcCommand implements TopologyAffectedCommand {
    public static final byte COMMAND_ID = 25;
 
-   private PublisherHandler publisherHandler;
```

Comment:
Whoops :)

Reply:
It wasn't breaking anything, just sitting there :)